about.html

```diff
@@ -0,0 +1,123 @@
+              <a href="#sponsors">Sponsor</a>
+              <a href="#">News</a>
+            </div>
+            <div class="campaign-container">
+              <button class="campaign-button">Chess Summit</button>
+            </div>
+          </div>
+        </nav>
+      </header>
+
+    <section class="about">
+        <img class="hamburger" id="hamb-button" src="images/hamburger.png" alt="">
+        <h1>GLOBAL SUMMIT OF CHESS 2022</h1>
+        <p>The Global Chess Summit brings together the community of
+          experts, academics, and activists who comprise the Chess
+          affiliate network in a different country every two.
+          This year, we hope to expand our invitation list including
+          organizations and individuals.
+        </p>
+        <div class="text-mail">
+          Please contact us per Email for any further questions about Chess Global Summit 2022!
+          <div class="about-mail">chesssummit2022@argen.org</div>
+        </div>
+      </section>
+    
+      <section class="about-2">
+        <h1>Chess Global Summit 2022 Logo</h1>
+        <div class="line-small"></div>
+        <h4>The logo of Chess Global Summit 2022 was decided through the logo
+          competition from 8. June to 7. July.</h4>
+        <div class="logo-box">
+          <img class="chess-logo" src="images/chess-logo.png" alt="logo image" />
+        </div>
+        <hr />
+      </section>
+    
+      <section class="about-3">
+        <h1>See The Past Chess Events</h1>
+        <div class="line-small"></div>
+        <h4>Take a look at last 2 Chess summits which took place in Budapest and Greece.</h4>
+        <div class="desktop-pic">
+          <div class="picture-1">
+            <div class="conf-cont1">
+              <div class="conf-text1">2020</div>
+              <div class="conf-text2">Chess Global Summit in Budapest</div>
+            </div>
+          </div>
+    
+          <div class="picture-2">
+            <div class="conf-cont1">
+              <div class="conf-text1">2021</div>
+              <div class="conf-text2">Chess Global Summit in Greece</div>
+            </div>
+          </div>
+        </div>
+      </section>
+    
+      <section id="sponsors">
+        <div class="sponsors">
+          <h1>Partner</h1>
+          <div class="line-small"></div>
+          <div class="sponsor-names">
+            <img src="images\sponsor1.png" alt="">
+            <img src="images\sponsor2.png" alt="">
+            <img src="images\sponsor3.png" alt="">
+            <img src="images\sponsor4.png" alt="">
+            <img src="images\sponsor5.png" alt="">
+          </div>
+        </div>
+    
+        <footer class="about-foot">
+          <div class="chess-footer-img">
+            <img src="images\chess-logo.png" alt="" />
+          </div>
+          <div class="footer-text">
+            2022 Chess Global Summit. Some Rights Reserved.
+          </div>
+        </footer>
+    
+      </section>
```

Comment:
[OPTIONAL] Kindly ensure that your project sticks as much as possible to the template provided. The color of this section in the desktop version is different from the original design. Kindly change this. :pray: :pray: Also pay attention please to the font color.

Your design
![image](https://user-images.githubusercontent.com/98363075/197097721-f3a6141b-bcdd-4bdc-8a75-eb3a4b4c9951.png)

Template design
![image](https://user-images.githubusercontent.com/98363075/197097812-d041ac76-59bc-4a49-9d68-82b9138647f8.png)